flight/drone/drone.py

```diff
@@ -0,0 +1,186 @@
+from dronekit import Vehicle
+import logging
+import time
+import threading
+from math import radians, sin, cos
+from pymavlink import mavutil
+
+from optical_flow_attribute import OpticalFlow
+from .. import constants as c
+from ..utils.timer import Timer
+
+class Drone(Vehicle):
+    """Interface to drone and its sensors.
+
+    Attributes
+    ----------
+    _id : int
+        A unique identifier for this drone.
+    """
+
+    def __init__(self, *args):
+        super(Drone, self).__init__(*args)
+        self._id = None
+        self._logger = logging.getLogger(__name__)
+
+        self._optical_flow = OpticalFlow()
+
+        # Allow us to listen for optical flow dat
+        @self.on_message('OPTICAL_FLOW')
```

Comment:
```suggestion
        @self.on_message(OPTICAL_FLOW_MESSAGE)
```

Reply:
The string here has to exactly match the name of the mavlink attribute. See https://mavlink.io/en/messages/common.html#OPTICAL_FLOW. on_message() is a dronekit.Vehicle method.